app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt

```diff
@@ -105,11 +104,26 @@ class MainFragment : Fragment() {
                     .navigate(MainFragmentDirections.actionMainFragmentToNewProfileFragment())
             }
         }
-
+        currentlyActiveProfile()
         setHasOptionsMenu(true)
         return binding.root
     }
 
+    private fun currentlyActiveProfile() {
```

Comment:
@Haid-Faiz I think you have incorrectly resolved the merge conflicts. This change has nothing to do with your PR.
I would suggest reverting the last changes you made, then merge the master code and then push the code in the PR.

Reply:
@rob729  I did it like.... after making changes like adding animation, i commited these changes then i pull the code of your master branch (upstream) then it gets merged in my local master branch  then after squashing the commits.... i forced push the code to my remote (origin) master branch then i created PR.

Did i do it in wrong way...???

Reply:
No. It seems to be right. Can you revert the recent commits and do this process again?
You can also try by first merging the master code, make the changes to the code, and then squash the commits.

Reply:
@rob729 okk